coordinator/src/db/user.rs

```diff
@@ -24,6 +28,8 @@ impl From<RegisterParams> for User {
             email: value.email.unwrap_or("".to_owned()),
             nostr: value.nostr.unwrap_or("".to_owned()),
             timestamp: OffsetDateTime::now_utc(),
+            // TODO: Consider allowing passing the FCM token in the registration request.
+            fcm_token: "".to_owned(),
```

Comment:
Is a user's fcm token final or will it change? I guess if the token is final you could pass it in the registration.

Reply:
Looks like it is not final, then I would not put it here: https://github.com/get10101/10101/pull/1227#discussion_r1316940179